code/src/main/kotlin/com/expediagroup/sdk/lodgingconnectivity/graphql/model/paging/PaginationControl.kt

```diff
@@ -0,0 +1,6 @@
+package com.expediagroup.sdk.lodgingconnectivity.graphql.model.paging
+
+data class PaginationControl @JvmOverloads constructor(
```

Comment:
Do we really need a wrapper for this? If so, let's name it `PageMetadata` or `PaginationMetadata`.

Reply:
Removed it!

We're passing the `cursor` and `pageSize` individually when needed - supporting that with `@JvmOverloads` to get multiple versions of the method. Also, I moved the `nextPageCursor` to the `PageInfo` model.

You can recheck the description, I've updated it.